.github/workflows/Tests.yml

```
@@ -19,6 +20,12 @@ jobs:
         ports:
           - 3306:3306
         options: --health-cmd="mysqladmin ping -h localhost -p3306" --health-interval=10s --health-timeout=5s --health-retries=5
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)